doc/tutorial/statistical_inference/finding_help.rst

```diff
@@ -13,9 +13,16 @@ ask on the `Mailing List <http://scikit-learn.org/stable/support.html>`_
 Q&A communities with Machine Learning practitioners
 ----------------------------------------------------
 
+  :Metaoptimize/QA:
```

Comment:
I think we may just need to drop this section. Certainly it does not belong first. The textbooks on machine learning will be increasingly outdated, and certainly are not pertinent to the Q&A communities headline.